include/okapi/api/units/RQuantity.hpp

```diff
@@ -192,6 +193,151 @@ constexpr bool operator>(const RQuantity<M, L, T, A> &lhs, const RQuantity<M, L,
   return (lhs.getValue() > rhs.getValue());
 }
 
+// Common math functions:
+// ------------------------------
+
+template <typename M, typename L, typename T, typename A>
+constexpr RQuantity<M, L, T, A> abs(const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<M, L, T, A>(std::abs(rhs.getValue()));
+}
+
+template <typename M, typename L, typename T, typename A>
+constexpr RQuantity<std::ratio_divide<M, std::ratio<2>>,
+                    std::ratio_divide<L, std::ratio<2>>,
+                    std::ratio_divide<T, std::ratio<2>>,
+                    std::ratio_divide<A, std::ratio<2>>>
+sqrt(const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<std::ratio_divide<M, std::ratio<2>>,
+                   std::ratio_divide<L, std::ratio<2>>,
+                   std::ratio_divide<T, std::ratio<2>>,
+                   std::ratio_divide<A, std::ratio<2>>>(std::sqrt(rhs.getValue()));
+}
+
+template <typename M, typename L, typename T, typename A>
+constexpr RQuantity<std::ratio_divide<M, std::ratio<3>>,
+                    std::ratio_divide<L, std::ratio<3>>,
+                    std::ratio_divide<T, std::ratio<3>>,
+                    std::ratio_divide<A, std::ratio<3>>>
+cbrt(const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<std::ratio_divide<M, std::ratio<3>>,
+                   std::ratio_divide<L, std::ratio<3>>,
+                   std::ratio_divide<T, std::ratio<3>>,
+                   std::ratio_divide<A, std::ratio<3>>>(std::cbrt(rhs.getValue()));
+}
+
+template <typename M, typename L, typename T, typename A>
+constexpr RQuantity<M, L, T, A> hypot(const RQuantity<M, L, T, A> &lhs,
+                                      const RQuantity<M, L, T, A> &rhs) {
+  return RQuantity<M, L, T, A>(std::hypot(lhs.getValue(), rhs.getValue()));
+}
+
+template <typename M, typename L, typename T, typename A>
```

Comment:
These methods should not be necessary. Internally, `std::max` uses comparison operators, which are compatible with RQuantity. This means the 
```cpp
template< class T > constexpr T max( std::initializer_list<T> ilist );
```
overload (which imo is quite useful) is also available.

Reply:
AFAIK this seems right. @Breq16 try removing these methods and compiling some code that used them.

Reply:
That seems to work! I'm kind of a noob when it comes to C++, so thanks for letting me know about this behavior.